build_runner_core/test/asset/lru_cache_test.dart

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
+@Timeout(Duration(seconds: 10))
```

Comment:
The annotation actually won't stop the endless execution, anyone knows how to fix it?

Reply:
The way timeouts work in this package is a bit weird - what it actually means is that the test must "make progress" in the timeout window. As long as it executes any code at all, the timeout resets. It only catches things that are stuck completely and not executing any async tasks.

Reply:
I see, thanks for the explanation..